tests/FSharp.AWS.DynamoDB.Tests/ConditionalExpressionTests.fs

```diff
@@ -104,6 +100,14 @@ type ``Conditional Expression Tests`` (fixture : TableFixture) =
         fun () -> table.PutItem(item, precondition = itemDoesNotExist)
         |> shouldFailwith<_, ConditionalCheckFailedException>
 
+    let [<Fact(Skip = "Not sure if this should be working")>] ``Item not exists failure should add conflicting item data to the exception`` () =
```

Comment:
this looks reasonable to me - don't have a prod env to hand so I guess that points the finget of blame at the simulators...